youtube_dl/extractor/zoom.py

```diff
@@ -54,15 +54,22 @@ def _real_extract(self, url):
         data = self._parse_json(self._search_regex(
             r'(?s)window\.__data__\s*=\s*({.+?});',
             webpage, 'data'), play_id, js_to_json)
-
+        try:
+            video_page = self._parse_json(self._download_webpage(
+                base_url + "nws/recording/1.0/play/info/" + data["fileId"], play_id),
+                play_id, js_to_json)
+        except Exception:
+            video_page = self._parse_json(self._download_webpage(
+                base_url + "nws/recording/1.0/play/share-info/" + data["fileId"], play_id),
+                play_id, js_to_json)
         return {
```

Comment:
This would have reduced the amount of code changes:
```suggestion
        data = video_page['result']

        return {
```